src/raydium/infos.ts

```diff
@@ -717,6 +717,22 @@ export class PoolInfoWrapper implements IPoolInfoWrapper {
     return apr;
   }
 
+  getTokenAAmount(tokenBAmount: bigint): bigint {
```

Comment:
Might need to add interface for wrapper?

Reply:
Okay, I'll add a optional interface.